### ISSUE TITLE
Insert modal does not include brands

### ISSUE THREAD

**Issue by Lulech23** [yukyhendiawan/icon-font-awesome#1]
First off, thanks for a great plugin! This really seems like it has the potential to be the best FontAwesome plugin for WordPress out there. 

However, when I started migrating my icons over from the official plugin to this one, I quickly ran into a problem: the inserter modal doesn't include icons from `fa-brands`. Or more specifically, it only includes icons from `fa-solid`. This would also exclude `fa-classic`, `fa-regular`, and `fa-sharp-solid`. It would be great if all classes could be included, as even editing a block as HTML and changing classes manually just results in a corrupt block.

**Comment by maltfield** [comment]
> even editing a block as HTML and changing classes manually just results in a corrupt block.

Yeah, and if we edit the `Additional CSS Class(es)` on the block, it gets added to the wrapper div -- not the `<i>` element as-needed.

For example, we wanted to change this

```
<div class="wp-block-yuky-icon-font-awesome"><div class="box-icon"><i class="fa-solid fa-credit-card" aria-label="Credit Card" style="font-size:40px;color:#333;--icon-font-awesome-hover-color:#333"></i></div></div>
```

To this

```
<div class="wp-block-yuky-icon-font-awesome"><div class="box-icon"><i class="fa fa-2x fa-fw black-icon fa-credit-card" aria-label="Credit Card" style="font-size:40px;color:#333;--icon-font-awesome-hover-color:#333"></i></div></div>
```

...but when we edited the `Additional CSS Class(es)` on the block, it produced this -- which didn't change the icon as we wanted

```
<div class="wp-block-yuky-icon-font-awesome"><div class="box-icon"><i class="fa-solid fa-credit-card" aria-label="Credit Card" style="font-size:40px;color:#333;--icon-font-awesome-hover-color:#333"></i></div></div>
```

That's a bit of a different feature from this (though one that would be more robust and provide a workaround for this issue and others), so I created a distinct ticket for it here:

 * https://github.com/yukyhendiawan/icon-font-awesome/issues/7
